contents/docs/reading-data.mdx

```diff
@@ -23,7 +23,7 @@ In the future, we'll [`freeze`](https://developer.mozilla.org/en-US/docs/Web/Jav
 
 ## Select
 
-ZQL queries start by selecting a table. There is no way to select a subset of columns; ZQL queries always return the entire row (modulo column permissions).
+ZQL queries start by selecting a table. There is no way to select a subset of columns; ZQL queries always return the entire row, if permissions allow it.
```

Comment:
Wait, we don't even have column permissions for select. Not sure what I was thinking here.